Arrowgene.Ddon.Server/Network/PingRequestPacketHandler.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Timers;
+using Arrowgene.Ddon.Shared.Entity;
+
+namespace Arrowgene.Ddon.Server.Network
+{
+    public abstract class PingRequestPacketHandler<TClient, TReqStruct, TResStruct> : RequestPacketHandler<TClient, TReqStruct, TResStruct>
+        where TClient : Client
+        where TReqStruct : class, IPacketStructure, new()
+        where TResStruct : ServerResponse, new()
+    {
+
+        private static readonly double TIMER_INTERVAL_MS = 60000; // 1 minute
+
+        private readonly Timer TimeoutCheckTimer;
```

Comment:
From what I just read up on the C# Disposable API, it is recommended to implement IDisposable when using disposable resources: https://learn.microsoft.com/en-us/dotnet/fundamentals/runtime-libraries/system-idisposable + https://stackoverflow.com/questions/41846853/timer-dispose-in-c-sharp

That means we should probably make this handler disposable and then clean up & call the dispose method / use the using syntax on disposable handlers so that on server shutdown all disposable resources are disposed of? Though we could also just filter out the handlers that implement IDisposable to then take care of it on our shutdown hook.
(but I am not an expert on C# idioms)